src/components/mainHeroSection/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC, ReactNode } from 'react';
+
+import { Box, Heading, Text } from '@chakra-ui/react';
+
+import Stack from '../stack';
+
+interface Props {
+  title: string;
+  text?: string;
+  bild?: ReactNode;
```

Comment:
```suggestion
  image?: ReactNode;
```
Let's code in English, please :)

Reply:
😅